api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -147,6 +147,39 @@ public async Task<IActionResult> AddAsync(ContentModel model)
         // return CreatedAtRoute("EditorContentFindById", new { id = content.Id }, new ContentModel(content));
     }
 
+    /// <summary>
+    /// Update content for the specified 'id'.
+    /// Publish message to kafka to index content in elasticsearch.
+    /// </summary>
+    /// <param name="model"></param>
+    /// <returns></returns>
+    [HttpPut("{id}")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Content" })]
+    public async Task<IActionResult> UpdateAsync(ContentModel model)
```

Comment:
Added this to the services so that they will use SignalR to inform users of updates.